lara-typescript/src/plugin-api/events.ts

```diff
@@ -29,4 +37,15 @@ export const events = {
    * Removes InteractiveAvailable event handler.
    */
   offInteractiveAvailable: (handler: IInteractiveAvailableEventHandler) => offInteractiveAvailable(handler),
+  /**
+   * Subscribes to InteractiveAvailable events. Gets called when any interactive changes its availablity state.
```

Comment:
This comment and the one below should be updated.

Reply:
Fixed